examples/medplum-patient-intake-demo/src/bots/core/test-data/intake-form-test-data.ts

```diff
@@ -12,9 +12,208 @@ export const intakePatient: Patient = {
   ],
 };
 
+export const intakeQuestionnaire: Questionnaire = {
+  resourceType: 'Questionnaire',
+  id: '58b7094f-4b04-46d3-9753-0272b1681a0c',
```

Comment:
remove id

Reply:
Ids are actually important here because they are being used to make references between test fixtures. Is there a better way to do it without ids?